### ISSUE TITLE
Combat feature list.

### ISSUE THREAD

**Issue by gjgfuj** [gjgfuj/Fighter#1]
As far as I can see:
- [ ] Networking
- [ ] Health bars. How?
- [ ] Characters can move around, duck, and jump?
- [ ] Have multiple hitboxes. (Head, body, legs?, each with a different knockback animation, and different stats, etc?)

Attacking:
- [ ] Players press buttons to activate attacks
- [ ] It plays an animation for x frames
- [ ] then an attack entity spawns.
  If the attack entity collides with one of the character's hitboxes:
- [ ] the attack entity despawns?
- [ ] the character's health depletes
- [ ] And an animation of them being knocked back plays, then their hitboxes are in new positions.
  Rinse and repeat.

**Comment by cdev-sci** [comment]
Don't forget about hitstung. Also while we will need multiple hitboxes per model to accurately resolve hits, different effects per hitbox are probably not necessary. For example, in Street Fighter attacks just have a property that dictates whether they hit HIGH,MID or LOW ,which is much easier to process than giving properties to hitboxes,

**Comment by sayi50** [comment]
Indeed, giving properties to the attacks that say which type of attack they are would be easier than splitting the hitboxes into 3 parts of the body. Just having a check on hit for that would be good enough.
